frontend/sozisel-app/src/components/SessionResultScreen/SessionResultEvents/EventResultDetails/Modules/QuizResultDetails/QuizResultViews/Charts/QuizResultChartsView.tsx

```diff
@@ -0,0 +1,91 @@
+import "./QuizResultChartsView.scss";
+
+import { CircularProgress, Typography } from "@material-ui/core";
+
+import ErrorAlert from "../../../../../../../utils/Alerts/ErrorAlert";
+import React from "react";
+import TotalAreaChart from "../../../../../../SessionResultSummary/TotalAreaChart";
+import TrendingUpIcon from "@material-ui/icons/TrendingUp";
+import { useQuizQuestionsSummaryQuery } from "../../../../../../../../graphql";
+import { useTranslation } from "react-i18next";
+
+export interface QuizResultChartsViewProps {
+  id: string;
+}
+export default function QuizResultChartsView({
+  id,
+}: QuizResultChartsViewProps): React.ReactElement {
+  const { t } = useTranslation("common");
+  const { loading, data } = useQuizQuestionsSummaryQuery({ variables: { id } });
+
+  if (loading) {
+    return (
+      <div className="QuizResultChartsView">
+        <CircularProgress />
+      </div>
+    );
+  }
+
+  if (data?.quizQuestionsSummary) {
+    const pointsChartData = data.quizQuestionsSummary.map((question) => {
```

Comment:
This should be inside a memo same as the variables below. Maybe for now the component does not have any interactions that would cause rerenders but once you add even a simple thing changing the state then this will blow up